src/Stratis.Bitcoin.IntegrationTests.Common/TestHelper.cs

```diff
@@ -50,8 +50,7 @@ public static void TriggerSync(CoreNode node)
 
         public static bool IsNodeConnected(CoreNode node)
         {
-            if (node.FullNode.ConnectionManager.ConnectedPeers.Any()) return true;
-            return false;
+            return node.FullNode.ConnectionManager.ConnectedPeers.Any(p => p.IsConnected);
```

Comment:
The question is, where this IsNodeConnected method is used, do we care about the node having some peers connected to it, or do we want to see that they're actually connected.

Reply:
I am not sure too be honest, perhaps only check that there was a connection, but then again if connection failed something went wrong

Reply:
As far as I have noticed, you can have Peers in the connectedPeers that are marked disconnected but stay in the list. That gives us timeouts when we check that a not is not connected in some integration tests (ex: reorg tests)

Reply:
>you can have Peers in the connectedPeers that are marked disconnected but stay in the list.

Do they stay in the list just a little longer and get removed or they don't get removed at all?
This is crucial if bad peers just stay in the list we have a bug, else it was just an expected race condition.